catalogue/webapp/components/WorkDetails/WorkDetails.js

```diff
@@ -93,7 +93,8 @@ type Props = {|
   iiifImageLocationCredit: ?string,
   iiifImageLocationLicenseId: ?LicenseType,
   encoreLink: ?string,
-  showSingleImageWorkPreview: boolean,
+  showWorkPreview: boolean,
+  showMultiImageWorkPreview: boolean,
```

Comment:
Why does this need to be a toggle?
We don't have another version of this, and it isn't in prod yet?